edx/edx-enterprise
# [ENT-633] Remove 'Discounted provided by...' text on programs landing page.

**Description:** We decided to remove the 'Discount provided by...' text on the program landing page because we don't currently have an easy way to show who provided the discount given the way we gather discount data from eCommerce today.

**JIRA:** [ENT-633](https://openedx.atlassian.net/browse/ENT-633)

**Installation instructions:** List any non-trivial installation 
instructions.

**Testing instructions:**

1. Apply a coupon to a program for an enterprise customer.
1. Go to the programs landing page and ensure that you can see a discount, but don't see any "Discount provided by..." text.
